lib/scala/runtime-version-manager-test/src/main/scala/org/enso/runtimeversionmanager/test/FakeEnvironment.scala

```diff
@@ -8,7 +8,7 @@ import org.enso.runtimeversionmanager.{Environment, FileSystem}
   * which points to an Enso installation inside the temporary directory
   * generated for the test.
   */
-trait FakeEnvironment { self: WithTemporaryDirectory =>
+trait FakeEnvironment { self: { def getTestDirectory: Path } =>
```

Comment:
That is correct.

I assumed it's not an issue as it is used inside of tests, so this one more reflection call is not really going to change much. But I guess having an interface for it will just be better.